docs/contributing/contributing.md

```diff
@@ -3,34 +3,76 @@ id: contributing
+ 1. Contribute through a GitHub Pull Request on the public [XSOAR Content Repository](https://github.com/demisto/content). Use this flow in the following scenarios:
+     - You are a Technology Partner or a developer contributing with *partner* supported new content.
+     - Your contribution is big and contains lots of different parts (Integrations, Scripts, Playbooks, Layouts, etc) that is likely to lead to a complex review process.
+     - You are proficient with GitHub.
+ 1. Contribute through a private GitHub repository: this is required if you are providing a **Premium** (aka Paid) Content Pack. The requirements are the same of supported Packs, and the contribution process is described [here](../integrations/premium_packs).
+
+This document describes the main flow that covers *supported* contributions and summarizes everything you must do before and after opening a Pull Request on GitHub to contribute your pack.
+
+If you are contributing *community* supported content, feel free to skip the rest of this document (although it's still recommended to read it to be aware of the best practices).
+
+## Contributor Guidelines
+
+Please read the following guidelines carefully: following them will maximize the chances for a fast, easy, and effective review process for everyone involved. If something is not clear, please don't hesitate to reach out to us via [Slack](http://go.demisto.com/join-our-slack-community) on the `#demisto-developers` channel.
+
+1. Begin by designing your contribution: we recommend to follow the [Design](../concepts/design) guidelines to identify what you want to build and make sure it is aligned with our best practices. Also check out the [Design Tutorial](../tutorials/tut-design).
+1. Make sure you have all the [Contributing Requirements](../contributing/contrib-requirements) satisfied.
+1. Setup a development environment by following the brief [Dev Setup Guide](dev-setup) or the more detailed [Tutorial](../tutorials/tut-setup-dev).
+1. Review the [Contribution Checklist](../contributing/checklist).
+1. Follow the [Content Pack format](packs-format) to build your contribution. [demisto-sdk init](https://github.com/demisto/demisto-sdk/blob/master/demisto_sdk/commands/init/README.md) will help you create it.
+1. Depending on the content entities you need to build, navigate to the specific section of this website for details. If you are creating Integrations and/or Automations, make sure that you:
+    * Use the proper  [Directory Structure](../integrations/package-dir). [demisto-sdk init](https://github.com/demisto/demisto-sdk/blob/master/demisto_sdk/commands/init/README.md) will help you create it. If working on existing code, beyond trivial changes, we require converting to this structure as it allows running linting and unit tests and provides a clearer review process.
```

Comment:
Not sure if I understand what you mean : this statement is true for both Integrations and Automations, so "Directory Structure" should be generic, not just "Integration Directory Structure". and "Integration and Automations Directory Structure" is too long IMO.

Reply:
There are two things mentioned here:
1. The hyperlink text (Directory Structure) is too generic (especially since we have the pack structure which is a bit similar), so I wanted to change it a bit.
2. The article itself has a confusing title, again - Directory Structure. Then U suggested the other one. I agree it's a bit long, but I want to differentiate this article from the pack structure one, after all, it explains only the integration/automation directory structure.